packages/flame/lib/src/effects2/controllers/pause_effect_controller.dart

```diff
@@ -0,0 +1,13 @@
+import 'duration_effect_controller.dart';
+
+/// A controller that keeps constant progress level over [duration] seconds.
```

Comment:
What does constant progress level mean?

Reply:
The `.progress` property remains constant over the duration. For example, you can make the effect stuck at `progress==0` or `progress==1`. Since the progress isn't changing, visually it will look as if the effect is paused.

Suggestions to improve the description?

Reply:
Aah, that makes sense. Maybe just add something like the example you just wrote here and it will make total sense?